detox/ios/Detox/MethodInvocation.m

```diff
@@ -80,6 +80,24 @@ + (id) getValue:(id)value withType:(id)type onError:(void (^)(NSString*))onError
         if (![value isKindOfClass:[NSDictionary class]]) return nil;
         return [MethodInvocation invoke:value onError:onError];
     }
+	if ([type isEqualToString:@"UIDeviceOrientation"])
```

Comment:
The main advantage of the remote invocation mechanism is that we don't need to add any other new call to the protocol

I believe we didn't make the invocation mechanism easily accessible for extension, and this is our fault.